src/onegov/town6/views/meeting.py

```diff
@@ -86,9 +141,67 @@ def view_meeting(
     }
 
 
+@TownApp.form(
+    model=Meeting,
+    name='edit',
+    template='form.pt',
+    permission=Private,
+    form=get_meeting_form_class,
+    pass_model=True
+)
+def edit_meeting(
+    self: Meeting,
+    request: TownRequest,
+    form: MeetingForm
+) -> RenderData | Response:
+
+    if not request.app.org.ris_enabled:
+        raise HTTPNotFound()
+
+    layout = MeetingLayout(self, request)
+
+    if form.submitted(request):
+        form.populate_obj(self)
+        request.success(_('Your changes were saved'))
+        return request.redirect(request.link(self))
+
+    form.process(obj=self)
+
```

Comment:
```suggestion

```
While you did use `pass_model=True`, you forgot to remove the `form.process()` call, this can mess up some things that rely on `on_request()`. We should ideally get rid of every manual `form.process` call in edit views, but we can improve this iteratively.

I think that this should fix the weird behavior with file uploads you're experiencing.